tests/core/pyspec/eth2spec/test/helpers/execution_payload.py

```diff
@@ -263,6 +267,69 @@ def compute_el_block_hash_for_block(spec, block):
         spec, block.body.execution_payload, block.parent_root, requests_hash)
 
 
+def build_empty_execution_payload_header(spec, state):
+    latest = state.latest_execution_payload_header
+    timestamp = spec.compute_timestamp_at_slot(state, state.slot)
+    execution_payload = build_empty_execution_payload(spec, state)
+
+    execution_payload_header = spec.ExecutionPayloadHeader(
+        parent_hash = latest.block_hash,
+        fee_recipient = spec.ExecutionAddress(),
+        state_root = latest.state_root,
+        receipts_root = spec.Bytes32(),
+        logs_bloom = spec.ByteVector[spec.BYTES_PER_LOGS_BLOOM](),
+        prev_randao = spec.Bytes32(),
+        block_number = 0,
+        gas_limit = 0,
+        gas_used = 0,
+        timestamp = timestamp,
+        extra_data = spec.ByteList[spec.MAX_EXTRA_DATA_BYTES](),
+        base_fee_per_gas = 0,
+        block_hash = compute_el_block_hash(spec, execution_payload, state),
```

Comment:
@jtraglia do I need to calc the hash of the random execution payload for `block_hash` or just put random Hash32 value?

Reply:
I think you can/should use `compute_el_block_hash` here, not a random value.